client/index.html

```diff
@@ -3,125 +3,126 @@
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-  <head>
```

Comment:
Our style guide only says:
'''## HTML Files
- Double quotes over single quotes for attributes
- Comment at closing of element where it closes via class i.e.
```HTML
<div class="row">
  ...
</div><!-- end: row -->
```
'''